.github/workflows/podPublish.yml

```diff
@@ -1,14 +1,10 @@
 name: Pod-Publish
 on:
   push:
-    branches:
-      # Push events to branches matching refs/heads/
-      - 'main'
-      - 'main_0.1'
-      - 'main_0.2'
     tags:
       - 0.[0-9]+.[0-9]+ # Run automatically on new tags matching a 0.N.M pattern
-      - 0.[0-9]+.[0-9]+_main*
+      - 0.[0-9]+.[0-9]+_main_0.1
```

Comment:
We can be even stricter and make this `0.1.[0-9]+_main_0.1` and the next one `0.2.[0-9]+_main_0.2`

Reply:
great call